equinox/nn/_normalisation.py

```diff
@@ -267,3 +267,117 @@ def __call__(
             return out
         else:
             return out, state
+
+
+class RMSNorm(Module):
+    r"""
+    A simplified version of LayerNorm which rescales the inputs, but does not center
+    them. Optionally applies a learned reweighting of the transformed array afterward.
+
+    Given an input array $x$, this layer computes
+
+    $$\frac{x}{\sqrt{\varepsilon + \frac{1}{n}\Vert x \Vert^2_2}} \gamma + \beta$$
+
+    where $\Vert x \Vert^2_2 = \sum_{i=1}^n x_i^2$, $n = \dim(x)$, and $\gamma$ is a
+    learned array with the same shape as $x$ if `use_weight=True`, or
+    $\gamma = 1$ if `use_weight=False`, as proposed in
+    [this paper](https://browse.arxiv.org/abs/2307.14995). `\beta` is an optional bias
+    term.
+
+    ??? cite
```

Comment:
Nit: this will need a new line here in order to render correctly in the docs.

i.e.
```python
??? cite

    foo
```

Reply:
Also, whilst we're here -- maybe worth emphasising that `use_{weight,bias}=False` is the default.

(Should we keep it as the default? I appreciate that's maybe more useful, but it's inconsistent with the other normalisation layers. WDYT?)

Reply:
Thanks for raising the point about the defaults. I feel its good to maintain consistency, especially with `LayerNorm`. Changed defaults to on.